pywinauto/controls/common_controls.py

```diff
@@ -3450,10 +3450,12 @@ def set_color(self, place_of_color, red, green, blue):
             raise RuntimeError('Incorrect range of color, must be from 0 to 255')
         if not (0 <= blue <= 255):
             raise RuntimeError('Incorrect range of color, must be from 0 to 255')
-        color = (red, green, blue)
-        str_color = str(color)
+        str_red = str(red)
+        str_green = str(green)
+        str_blue = str(blue)
+        color = str_red + str_green + str_blue

```

Comment:
This is very strange decision. I think you smoke some drugs. :)
3-tuple with integers is OK. Why string?